lib/icingadb/redisconnection.hpp

```diff
@@ -158,14 +162,16 @@ namespace icinga
 		String m_Password;
 		int m_DbIndex;
 
+		static unsigned int m_PriorityOrder;
+
```

Comment:
Put it into m_Queues after Writes, name it NextWriteOrder and comment it as the other members.

Reply:
I said _after Writes_.